### ISSUE TITLE
Accounting of CPU time treats live and exited processes differently

### ISSUE THREAD

**Issue by graeme-a-stewart** [HSF/prmon#35]
Currently prmon CPU accounting separates user/system time measurements into two - the `utime` and `stime` for running processes and the `cutime` and `cstime` for exited child processed. When looking at the accounting for parents whose children exit this leads to some decreases in the `u/stime` that look strange, e.g.,

```
[root@aca2c5b831e5 tests]# cat prmon.txt 
Time		VMEM	PSS	RSS	Swap	rchar	wchar	rbytes	wbytes	utime	stime	cutime	cstime	wtime	rx_bytes	rx_packets	tx_bytes	tx_packets
1524239875	92048	2078	9648	0	5280	79	0	0	3.94	0.09	0	0	1	0	0	0	0
1524239877	92048	2078	9648	0	5280	79	0	0	11.93	0.17	0	0	3	0	0	0	0
1524239879	92048	2078	9648	0	5280	79	0	0	19.87	0.38	0	0	5	0	0	0	0
1524239881	92048	2078	9648	0	5280	79	0	0	27.86	0.56	0	0	7	0	0	0	0
1524239883	92048	2078	9648	0	5280	79	0	0	35.92	0.64	0	0	9	0	0	0	0
1524239885	23012	1491	3252	0	5280	79	0	0	20.64	0.44	19.54	0.29	11	0	0	0	0
1524239887	23012	1491	3252	0	5280	79	0	0	22.59	0.48	19.54	0.29	13	0	0	0	0
1524239889	23012	1491	3252	0	5280	79	0	0	24.58	0.5	19.54	0.29	15	0	0	0	0
1524239891	23012	1491	3252	0	5280	79	0	0	26.55	0.54	19.54	0.29	17	0	0	0	0
1524239893	23012	1491	3252	0	5280	79	0	0	28.51	0.59	19.54	0.29	19	0	0	0	0
```

See that `utime` and `stime` drop at 1524239885.

I think a more consistent accounting would be to sum `u` and `s` time for each parent and exited children together, which should give nice monotonic rising curves for these values.

**Comment by graeme-a-stewart** [comment]
A more consistent *test* for this is supported in #34

**Comment by graeme-a-stewart** [comment]
Fixed in both #36 and #37 (#37 merged in).
